tpot/builtins/one_hot_encoder.py

```diff
@@ -147,7 +144,8 @@ class OneHotEncoder(BaseEstimator, TransformerMixin):
     categorical_features: "all" or array of indices or mask
         Specify what features are treated as categorical.
 
-        - 'all' (default): All features are treated as categorical.
+        - 'all': All features are treated as categorical.
+        - 'auto' (default): Select only features that have less than 10 unique values.
```

Comment:
I think the # of unique values for the auto selection should be parameterized for the OneHotEncoder class, with a default of 10.